azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -162,6 +162,11 @@ public FlowRunnerManager(final Props props,
     this.executionDirectory = new File(props.getString("azkaban.execution.dir", "executions"));
     if (!this.executionDirectory.exists()) {
       this.executionDirectory.mkdirs();
+      // setting sticky bit on this.executionDirectory to allow cleanup thread to delete all
+      // user-generated files/directories
+      logger.info("Creating subprocess to run shell command: chmod g+s "
+          + this.executionDirectory.toString());
+      Runtime.getRuntime().exec("chmod g+s " + this.executionDirectory.toString());
```

Comment:
This is not sticky bit, is it? This is setgid. Right?

see
https://docs.oracle.com/cd/E19683-01/806-4078/secfiles-69/index.html
"When setgid permission is applied to a directory, files that were created in this directory belong to the group to which the directory belongs, not the group to which the creating process belongs. Any user who has write and execute permissions in the directory can create a file there. However, the file belongs to the group that owns the directory, not to the user's group ownership.
"